src/test/ui/methods/issues/issue-90315.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
```

Comment:
I'm a little confused -- should this be perhaps marked as `known-bug` or something like that? Otherwise this seems to be a test case for an example that is currently *not* good, which seems like it should at least have a comment as to what is being tested (it is not at all clear from just this file without reading the issue, which seems suboptimal to me).

Reply:
It used to say “the method `rev` exists for type `usize`”, which is false. Now, it says that “usize is not an iterator”, which is true.

I’ll add a comment to the file when I’m in front of a computer again.